### ISSUE TITLE
Config `dbUrl` based on env variable?

### ISSUE THREAD

**Issue by delaneyj** [adelsz/pgtyped#261]
I have a situation where there are multiple pg instances that have to be connected to.

Would like something like 

config.json
```json
{
  ...
  "dbUrl": "{{process.env.FOO_PG_CONN}}", 
  ...
}
```

Is there a way to support this?

**Comment by adelsz** [comment]
Currently this isn't supported. As a temporary workaround you can supply the DB config using the PG env vars `PGUSER`, `PGDATABASE`, etc..

**Comment by delaneyj** [comment]
In my case have two completely separate databases I'm trying to convert between.

**Comment by bradleyayers** [comment]
I've had similar friction and ended up doing something super hacky…

Added a `package.json` script:

```
"dev-pgtyped": "DOTENV_CONFIG_PATH=config/development node -r dotenv/config -r ts-node/register/transpile-only -r ./src/bin/pgTypedEnv.ts $(FORCE_COLOR=0 yarn bin pgtyped) -c pgtyped.json"
```

Contents of`./src/bin/pgTypedEnv.ts`:

```ts
Object.assign(process.env, {
  PGHOST: process.env.DATABASE_HOST,
  PGPASSWORD: process.env.DATABASE_ROOT_PASSWORD,
  PGUSER: process.env.DATABASE_ROOT_USER,
  PGDATABASE: process.env.DATABASE_NAME,
  PGPORT: process.env.DATABASE_PORT,
});
```

The simplest thing as a consumer would be to just let me use a `.js` instead of `.json` for the config file.

**Comment by adelsz** [comment]
> The simplest thing as a consumer would be to just let me use a .js instead of .json for the config file.

@bradleyayers, js file configs are already supported, we are doing a regular require call for the config:
https://github.com/adelsz/pgtyped/blob/bcdd309b6148bde86fe70ef9e7ea0b2b1feafca3/packages/cli/src/config.ts#L96

**Comment by adelsz** [comment]
There is no good reason why `DBURL` can't be passed through an env. It should be very straightforward to add, will try to add it to next release.

**Comment by adelsz** [comment]
Fixed in v0.11.0. You can now use the `--uri` flag or the `DBURI` env var.
